app/forms/access_form.rb

```diff
@@ -1,15 +1,26 @@
 # frozen_string_literal: true
 
 # A form object for updating the rights on an item
+# @abstract
 class AccessForm
   extend ActiveModel::Naming
-
-  # @param [Cocina::Models::DRO]
-  def initialize(model, apo)
+  # @param [Cocina::Models::DRO, Cocina::Models::Collection] model
+  def initialize(model, default_rights: nil)
```

Comment:
Can this be `default_rights: 'citation-only'`? Then line 10 can be `@default_rights = default_rights`

Reply:
no, because they may pass in `default_rights: nil` and we still want the instance variable to get a value.